test/dynamo/test_export.py

```diff
@@ -2333,6 +2333,37 @@ def f(x, y):
                 preserved = True
         self.assertTrue(preserved)
 
+    @torch._dynamo.config.patch(
+        dynamic_shapes=True,
+        capture_dynamic_output_shape_ops=True,
+        specialize_int=True,
+        capture_scalar_outputs=True,
+    )
+    def test_export_preserve_constraints_as_metadata_tensor(self):
+        from torch._export.constraints import constrain_as_value
```

Comment:
Can you move it at the top of file? We recently ran into circular import issue, so i wanted to make sure it doesn't happen again?